ci_tools/bumpversion.sh

```diff
@@ -1,5 +1,24 @@
-bump2version patch --message "Bump version: {current_version} to {new_version} [ci skip]"
-git config --local user.email "engineering@finbourne.com"
-git config --local user.name "finbourne-bot-public"
-git push https://finbourne-bot-public:${gh_token}@github.com/finbourne/lusid-python-tools.git HEAD:$TRAVIS_BRANCH
-echo "Making commit with incremented version"
\ No newline at end of file
+#!/bin/bash
+
+last_commit_message=$(git log -1 HEAD --pretty=format:%s)
+
+# Find next version
+
+api_version=$(cat __version__.py | grep __version__ |  awk '{split($0, a, "="); print a[2]}' | tr -d ' "')
```

Comment:
i think this block can be moved to line 18 (where it is used)

Reply:
Done